front/src/applications/operationalStudies/components/ImportTrainSchedule/ImportTrainScheduleConfig.tsx

```diff
@@ -78,7 +78,7 @@ export default function ImportTrainScheduleConfig(props: ImportTrainScheduleConf
         const stepsWithDuration = trainSchedule.steps.map((step) => {
           // calcul duration in seconds between step arrival and departure
           const duration = Math.round(
-            (new Date(step.arrivalTime).getTime() - new Date(step.departureTime).getTime()) / 1000
```

Comment:
Can you add a comment here to explain why the arrivalTime is before the departureTime ?